metadata_mapper/mappers/mapper.py

```diff
@@ -1601,21 +1596,15 @@ def compose_repo_data(repo):
                 if isinstance(date_source, dict):
                     date_source = [date_source]
 
-                # make_datetime is not implemented in rikolti and this should
-                # fail if executed. Amy thinks this is unused cruft, and she
-                # is very curious to know about any records that have this data
                 dates_start = [
-                    make_datetime(dt.get('begin', None))  # noqa: F821
-                    for dt in date_source if isinstance(dt, dict)]
+                    parser.parse(dt.get('begin', None)).strftime("%Y-%m-%d")
```

Comment:
Yeah, thanks. I'll take a look at this.

Reply:
@lthurston since this PR updates so much of `mapper.py`, I'm thinking we should pull it in sooner rather than later and move the implementation of `make_datetime` into a separate PR to finish out the UCSD Blacklight work.

Reply:
This took a little longer than I thought it would, but I think it's good to go now. You should scrutinize it though. I've simplified the code and removed the data exception logging. If you want the logging back it's a quick change.

Reply:
Actually -- there's still a little bug. 🐛  I'll fix it before the end of the day.

Reply:
Ok, I think it's fixed!

Reply:
Thanks Lucas! Looks good to me. I'm merging this in now :)